catalog/core/catalog-core-ftp/src/main/java/ddf/catalog/core/ftp/UserManagerImpl.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.core.ftp;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.ftpserver.ftplet.Authentication;
+import org.apache.ftpserver.ftplet.Authority;
+import org.apache.ftpserver.ftplet.FtpException;
+import org.apache.ftpserver.ftplet.User;
+import org.apache.ftpserver.ftplet.UserManager;
+import org.apache.ftpserver.usermanager.impl.BaseUser;
+import org.apache.ftpserver.usermanager.impl.ConcurrentLoginPermission;
+import org.apache.ftpserver.usermanager.impl.TransferRatePermission;
+import org.apache.ftpserver.usermanager.impl.WritePermission;
+
+/**
+ * Overrides the FTP authentication so that we can rely on DDF auth instead. If our authentication
+ * fails, a {@link org.apache.ftpserver.ftplet.FtpletResult#DISCONNECT} signal will be sent to the client.
+ */
+public class UserManagerImpl implements UserManager {
+
+    private static final String UPLOAD_DIRECTORY = "";
+
+    private static final String DEFAULT_USER = "default";
+
+    private BaseUser user;
+
+    public UserManagerImpl() {
+    }
+
+    public synchronized void save(User usr) throws FtpException {
+    }
+
+    public void delete(String usrName) throws FtpException {
+    }
+
+    public String[] getAllUserNames() {
+        ArrayList<String> ulst = new ArrayList<String>();
+        ulst.add(DEFAULT_USER);
+
+        Collections.sort(ulst);
```

Comment:
The ArrayList to array conversion seems unnecessary. Could replace method contents with:
return new String[] {DEFAULT_USER}; 
if that the intent here.